addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1278,6 +1272,18 @@ registerModel({
         recoveryDelay: attr({
             default: 3000,
         }),
+        /**
+         * Object that represents a set of peerTokens, used to track which calls are outgoing,
+         * which is used when attempting to recover a failed peer connection by
+         * inverting the call direction.
+         *
+         * Key: token
+         * Value: truthy when set
+         */
+        rtcOutgoingCalls: many('RtcOutgoingCall', {
+            inverse: 'rtc',
+            isCausal: true,
+        }),
```

Comment:
Looks likes "is outgoing" should be flag on a more generic "call" model instead?

Reply:
Or more generally speaking, the first model to make in RTC should be `peer`, with `token` as identifying?
It looks all other models that will also use `token` as identifying should be just be related from peer?

Reply:
Done by turning all of this into `RtcSession/isOutgoing`. `token` is actually equivalent to `RtcSession/id` (= id of `RtcSession`), so all is on `RtcSession` when identified by `token`/`peer`.